root_VS2010/programs/C#/10_Build_WebApp_sample.bat

```diff
@@ -13,7 +13,7 @@ set ORG_PATH=%PATH%
 @rem --------------------------------------------------
 @rem Get the path to the executable file.
 @rem --------------------------------------------------
-set CURRENT_DIR=%~dp0
+set CURRENT_DIR="%~dp0"
```

Comment:
@daisukenishino 

You said as following in [issue #23](https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/23).
># Requirement
>We will stop the root_VS2010's update.
>Because some of .NET Framework will be no longer supported at January 12, 2016.
>http://blogs.msdn.com/b/dotnet/archive/2014/08/07/moving-to-the-net-framework-4-5-2.aspx
>
>Therefor, after it will be released by setting the 01-71tag,
>  to remove the root_VS2010 in the develop branch.

The update of files in root_VS2010 has stopped?Or not?
Because you said "after it will be released by setting the 01-71 tag", I thought that the update of root_VS2010 has stopped.